deploy/Dockerfile

```diff
@@ -1,6 +1,5 @@
 FROM registry.access.redhat.com/ubi7-minimal
```

Comment:
Do we actually need this file anymore? I think it is a leftover from the early days. And the FROM image is outdated anyway

Reply:
I actually thought one of you would use it locally

Reply:
Yes, in the past. But even locally let's use the others.

Reply:
i deleted it. But we still need the clowdapp.yaml file in deploy/?

Reply:
You have to ask @lahavyuv86 He seems to have made changes to it recently